build-in-container/scripts/setup.sh

```diff
@@ -62,24 +62,37 @@ check_specs() {
 # enable custom-repo.repo to install RPMs from
 setup_custom_repofile() {
     echo "------------ Setting up custom repofile ------------"
-    echo -e "\n" >> $MARINER_BASE_DIR/toolkit/resources/manifests/package/local.repo
-    cat $RPM_repo_file >> $MARINER_BASE_DIR/toolkit/resources/manifests/package/local.repo
-    echo -e "\n" >> $MARINER_BASE_DIR/toolkit/resources/manifests/package/local.repo
+    for repo_file in $(echo $RPM_repo_file | tr "," "\n")
+    do
+        # append baseurl from $repo_file to $PACKAGE_LIST_URL to use them for downloading toolchain RPMs
+        PACKAGE_URL_LIST+=" "
+        PACKAGE_URL_LIST+=$(cat $repo_file | grep baseurl | cut -d '=' -f 2)
```

Comment:
I'm not sure what the right path here is... I like this idea, but it also bings an edge case. By setting `PACKAGE_URL_LIST` we effectively remove PMC from our list. If the user intentionally doesn't want PMC that is fine, but if they just want to add an extra repo on top of PMC this is a problem.

We can scrape the contents of the existing PACKAGE_URL_LIST from our makefile if we want... but that is getting complicated.

Reply:
I am only appending the new baseURLs here, and appending a " " between the different entries

Reply:
Unfortunately by setting this variable prior to calling the makefile, it will pre-populate the value for `PACKAGE_URL_LIST`, which means the default (PMC) is never set by Make. 

We do have a mechanism in the makefile to output its default/calculated values, maybe it can be used here to pre-populate the value? `default=$(make printvar-PACKAGE_URL_LIST 2>/dev/null)`